### ISSUE TITLE
Example of Combine – sync as many devices as you want over wifi, lan or MIDI ?

### ISSUE THREAD

**Issue by DavidOpgh** [moritzvieli/rocketshow#66]
Is there any kind of documentation / example of this feature of Rocketshow?

I have a Pi 3 I'd like to try this out with.

**Comment by moritzvieli** [comment]
@DavidOpgh No, we need to create some proper documentation here as well. :)

But in the end, it's really simple: The devices need to be able to connect to each other in a network. It's easiest to connect them with a switch and a network cable. Then you can a master node and add all slave nodes in its network settings. As soon as you play/stop/pause compositions on the master device, the slave nodes will be triggered as well.

Connection is also possible over wifi instead of wired connections. We'd recommend a good access point (and not letting a Raspberry act as an access point). Connecting the devices over wifi has to be done by ssh-ing into the Raspberry Pi however with Raspberry Pi OS and is currently not supported with settings over Rocket Show.

Alternatively, you can trigger the devices over MIDI. In this case, each device requires a MIDI device.
